fetchComponent.py

```diff
@@ -148,8 +148,8 @@ def postProcessFootprint(footprint):
     refPos = topMiddle(bbox) + pcbnew.wxPoint(0, -footprint.Reference().GetTextHeight())
     valuePos = bottomMiddle(bbox) + pcbnew.wxPoint(0, +footprint.Reference().GetTextHeight())
 
-    footprint.Reference().SetPosition(refPos)
-    footprint.Value().SetPosition(valuePos)
+    footprint.Reference().SetPosition(pcbnew.VECTOR2I(refPos))
+    footprint.Value().SetPosition(pcbnew.VECTOR2I(valuePos))
```

Comment:
Does this change continue to work with KiCad 6? It might be preferable to maintain KiCad 6 for at least a year after it was replaced by KiCad 7.  But that is just my suggestion.

Reply:
that might be the thing, to be honest i just changed it so that works for me

Reply:
There is https://github.com/yaqwsx/pcbnewTransition that monkeypatches older KiCAD API to match the newest one. You just need to use `from pcbnewTransition import pcbnew` instead of `import pcbnew`.

Reply:
At this point KiCad 8 is imminent, and we're about a month short of a year since KiCad 7 was released. 

That being said, it would be no problem to just wrap this in a try/except and just catch the error that pcbnew throws if you use the legacy API.